lib/sidekiq/cron/views/cron.erb

```diff
@@ -63,7 +63,7 @@
         <tr>
           <td class="<%= klass %>"><%= t job.status %></td>
           <td class="<%= klass %>">
-            <a href="<%= root_path %>cron/namespaces/<%= @current_namespace %>/jobs/<%= escaped_job_name %>" title="<%= job.description %>">
+            <a href="<%= root_path %>cron/namespaces/<%= job.namespace %>/jobs/<%= escaped_job_name %>" title="<%= job.description %>">
```

Comment:
So maybe we can remove the `@current_namespace`? Not sure if this is used now.

Reply:
@markets `@current_namespace` is still used for actions like `Enqueue All`, `Enable All`, `Disable All`, and `Delete All`, which are applied at the namespace level.

Reply:
Can't we use `job.namespace` also on those actions?

Reply:
No, because those actions are applied to the requested namespace, which in our case can be `*`. If the namespace is `*`, it should enqueue all jobs, regardless of their specific namespace.